Update docs regarding the docker run suggestion issue (#132)
harvester/docs: docs/rancher/virtualization-management.md

For Harvester v0.3.0 and above, virtualization management with the multi-cluster management feature will be supported using Rancher v2.6.x.

> 1. As a prerequisite, you will need to create a Rancher server with `v2.6.3` or above for Harvester `v1.0.0` integration. In production, use one of the following guides to deploy and provision Rancher and a Kubernetes cluster in the provider of your choice:

Reviewer: ```suggestion
1. As a prerequisite, Harvester `v1.0.0` integration requires Rancher server `v2.6.3` or above. In production, use one of the following guides to deploy and provision Rancher and a Kubernetes cluster with the provider of your choice:
```
